lightkurve/io/tasoc.py

```diff
@@ -0,0 +1,48 @@
+"""Reader for A PSF-Based Approach to TESS High Quality Data Of Stellar Clusters (PATHOS) light curve files.
+
+Website: https://archive.stsci.edu/hlsp/pathos
+A product description file wasn't obvious on the MAST website
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+
+from .generic import read_generic_lightcurve
+
+
+def read_tasoc_lightcurve(filename,
+                            flux_column="FLUX_RAW",
+                            quality_bitmask="default"):
+    """Returns a `TessLightCurve`.
+
+    Parameters
+    ----------
+    filename : str
+        Local path or remote url of TASOC light curve FITS file.
+    flux_column : 'flux_RAW' - this contains the T'DA extracted lightcurve,
+    with no corrections applied to the raw light curves. Corrected lightcurves 
+    may be a thing in the future as there is a flux_corr column.
```

Comment:
To enable the documentation to be generated automatically, it is important that we stick to the "numpydoc" docstring standard, which looks like this:
```
parameter : type
    One or more indented lines explaining the role of the parameter.
```

Suggested fix:
```suggestion
    flux_column : str
        Column that will be used to populate the flux values.
        By default, "FLUX_RAW" is used. It contains the T'DA extracted lightcurve,
        with no corrections applied to the raw light curves. Corrected lightcurves 
        may become available in the future.
```

Reply:
Changed to reflect suggestion above